lib/matplotlib/tests/test_axes.py

```diff
@@ -7280,6 +7280,20 @@ def test_secondary_repr():
     assert repr(secax) == '<SecondaryAxis: >'
 
 
+@image_comparison(['axis_options.png'], remove_text=True, style='mpl20')
+def test_axis_options():
+    fig, axes = plt.subplots(2, 3)
+    for i, option in enumerate(('scaled', 'tight', 'image')):
+        axes[0, i].plot((1, 2), (1, 3.2))
```

Comment:
What's thd difference between the top and bottom tests?

Reply:
Aspect ratio.

Reply:
Is that a relevant parameter to test? I would have expected that the code paths are the same.

If it is, please add comments á la `# aspect ratio > 1` to both parts.

Reply:
Is the aspect ratio coming from just the autoscaling on the line?

Testing both of these seems relevant, the Do What I Mean code tends to be full of (and grow) special cases so (low fidelity) fuzz testing is reasonable.

Reply:
Yes, that is my impression. It will at least not consider a large circle.